modules/infinite-scroll/infinity.css

```diff
@@ -18,6 +16,103 @@
 	padding: 6px 16px;
 }
 
+/**
+ * CSS Spinner Styles
+ */
```

Comment:
Removed the `jquery.spin` library and replaced the spinner with a purely CSS solution.

**Potential regression**: The original implementation allowed for re-coloring the spinner based on the current styles of the page. The new implementation doesn't easily allow of this feature and relies on the spinner being visible on most backgrounds. If this is a mission critical feature that needs to be retainer, please let me know - I just didn't want to spend larger amount of time on building this out without confirmation.

Reply:
That makes sense. How does the CSS solution currently look like on default themes?

Reply:
@jeherve It looks like this:
![image](https://user-images.githubusercontent.com/433570/77076670-6d01e980-69f4-11ea-934f-8668fa42aec6.png)

All default themes have white or in the case of `twentytwenty` yellow-ish background, so the black spinner works there. There is also `outline: 1px solid white` making sure the spinner is visible (though not pretty) on black and very dark backgrounds.

![image](https://user-images.githubusercontent.com/433570/77076553-4348c280-69f4-11ea-80a5-5e4ef3ecb3ad.png)

I have considered using [`mix-blend-mode: difference`](https://developer.mozilla.org/en-US/docs/Web/CSS/mix-blend-mode) to make it stand out regardless of the background color, but blend modes [are not applied](https://stackoverflow.com/questions/46358881/why-dont-css-blend-modes-work-when-applied-against-the-document-body) on the base layer in Chrome, which is a show stopper for spinners.

Reply:
This should be fine, thank you!